src/main/java/frc/robot/subsystems/StorageSubsystem.java

```diff
@@ -11,8 +11,8 @@
 
 public class StorageSubsystem extends SubsystemBase {
 
-  private final double SAFE_SPEED = 0;
-  private final int SAFE_COOLDOWN_MS = 1000;
+  private final double SAFE_SPEED = 1;
+  private final int SAFE_COOLDOWN_MS = 0;
```

Comment:
This pretty much invalidates the entire purpose of the SafeSP class. If you don't want the current monitoring, just use the VictorSP class.